### PR TITLE
Tried to generalize things so that loading is easier with a different dataset (like data of neurospin)

### PR DESCRIPTION
**Pull request by chaithyagr**

### CODE REVIEW

File: fastmri_recon/config.py [hunk 1]
```diff
@@ -3,9 +3,9 @@
 import os
 
 FASTMRI_DATA_DIR = os.environ.get('FASTMRI_DATA_DIR', '/media/Zaccharie/UHRes/')
-OASIS_DATA_DIR = os.environ.get('OASIS_DATA_DIR', '/media/Zaccharie/UHRes/OASIS_data')
+OASIS_DATA_DIR = os.environ.get('OASIS_DATA_DIR', '/volatile2/')
 LOGS_DIR = os.environ.get('LOGS_DIR', './')
-CHECKPOINTS_DIR = os.environ.get('CHECKPOINTS_DIR', './')
+CHECKPOINTS_DIR = os.environ.get('CHECKPOINTS_DIR', '/volatile2/model_checkpoints')
```
Comment (chaithyagr) [inline fastmri_recon/config.py:8]: Ahhh this is bad, I will remove this.

File: fastmri_recon/evaluate/scripts/paper_eval_oasis.py [hunk 2]
```diff
@@ -39,8 +39,8 @@ def evaluate_paper_oasis(n_samples=200):
         val_split=0.1,
     )
     val_gen_mask = train_gen.val_sequence
-    n_train = 1000
-    n_val = n_samples
+    n_train = min(1000, len(train_gen.filenames))
```
Comment (chaithyagr) [inline fastmri_recon/evaluate/scripts/paper_eval_oasis.py:42]: This is a minute update here to ensure that we dont cross the number of files

File: requirements.txt [hunk 3]
```diff
@@ -1,11 +1,11 @@
 numpy >= 1.18
-h5py<2.11.0,>=2.10.0
-scikit-image
+h5py>=2.10.0
```
Comment (chaithyagr) [inline requirements.txt:2]: This will help for using tensorflow 2.5

File: requirements.txt [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
+scikit-image==0.17.2
```
Comment (chaithyagr) [inline requirements.txt:3]: Scikit image must be <=0.17.2 .. I will get another PR for this soon, but this is a good first fix.

File: fastmri_recon/data/sequences/oasis_sequences.py [hunk 5]
```diff
@@ -17,7 +17,8 @@ def _get_subject_from_filename(filename):
     subject_id = re.findall(r'OAS3\d{4}', base_name)[0]
     return subject_id
 
-class Oasis2DSequence(Sequence):
+
+class NIFTY2DSequence(Sequence):
```
Comment (zaccharieramzi) [inline fastmri_recon/data/sequences/oasis_sequences.py:21]:
```suggestion
class NIFTI2DSequence(Sequence):
```

I think the file format is `nifti` not nifty

File: fastmri_recon/evaluate/scripts/paper_eval_oasis.py (hunk 2 above)
Comment (zaccharieramzi) [reply fastmri_recon/evaluate/scripts/paper_eval_oasis.py:42]: makes sense

File: fastmri_recon/config.py (hunk 1 above)
Comment (zaccharieramzi) [reply fastmri_recon/config.py:8]: yes thanks !

**Review by zaccharieramzi — CHANGES_REQUESTED** [review]
some small comments

**Pull request closed without merging**